chain-core/src/tx/data/address.rs

```diff
@@ -27,14 +29,8 @@ pub enum ExtendedAddr {
 impl ExtendedAddr {
     fn get_string(&self, hash: TreeRoot) -> String {
         let checked_data: Vec<u5> = hash.to_vec().to_base32();
-        match crate::init::CURRENT_NETWORK {
-            crate::init::network::Network::Testnet => {
-                bech32::encode("crtt", checked_data).expect("bech32 crmt encoding")
-            }
-            crate::init::network::Network::Mainnet => {
-                bech32::encode("crmt", checked_data).expect("bech32 crmt encoding")
-            }
-        }
+
+        bech32::encode(get_bech32_human_part(), checked_data).expect("bech32 crmt encoding")
```

Comment:
`.expect("bech32 encoding")`

I guess this shouldn't ever panic?

Reply:
hardly not panic,
unwrap will be enough

Reply:
`expect` is fine (it's the same as unwrap, but will print a more meaningful panic) -- I just mean that there could be a comment above explaining why it wouldn't panic (or perhaps a compile-time assertion that proves it wouldn't panic https://github.com/nvzqz/static-assertions-rs#usage )

Reply:
got it.
add more message